home/.bin/create-hotfix

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+require 'shellwords'
+require 'tracker_api'
+require 'yaml'
+require 'active_support'
+
+### What it does
+# We created a script that will create a bug pivotal story and start a hotfix branch for you. All you need to do is pass a story name.
+
+# Setting it up
+#   If you want to use it, you have to add your Pivotal API Key as an
+#   environment variable. On my account, I put it in ~/.zsh/control-rails.zsh.
+#   (but all files in ~/.zsh our loaded so put it any file in that directory)
+#
+#   This script can also detect if you are "hitched" with someone using the
+#   hitch gem (it reads ~/.hitchrc). If you don't hitch the Pivotal card will
+#   be assigned to $USER (note your Pivotal user needs to match your unix user
+#   in this case)
+
+# i.e.
+#    export PIVOTAL_TOKEN="TOKEN HERE"
+#    export PIVOTAL_PROJECT_ID="some_number_here"
+#
+#    You can find your token at the bottom of the User Profile page on Pivotal,
+#    which you can get to by clicking your name in the top right corner.
+#
+# How you'd use it
+#     i.e. script/create-hotfix 'Do The Needful'
```

Comment:
is this still `script/create-hotfix` or being in `.bin` is it just `create-hotfix`?